packages/common/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "@botpress/common",
```

Comment:
**tought:** I think I'd prefer `@botpresshub/common` since this package is common to integrations only

Reply:
Good idea